src/main/java/uk/org/openbanking/datamodel/fund/OBFundsConfirmationConsentResponse1Data.java

```diff
@@ -35,239 +34,250 @@
 /**
  * OBFundsConfirmationConsentResponse1Data
  */
-@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2021-05-20T10:02:34.555654+01:00[Europe/London]")
-
+@javax.annotation.Generated(value = "org.openapitools.codegen.languages.SpringCodegen", date = "2021-11-18T09:57:44.731259Z[Europe/London]")
 public class OBFundsConfirmationConsentResponse1Data {
-  @JsonProperty("ConsentId")
-  private String consentId;
+    @JsonProperty("ConsentId")
+    private String consentId;
+
+    @JsonProperty("CreationDateTime")
+    @org.springframework.format.annotation.DateTimeFormat(iso = org.springframework.format.annotation.DateTimeFormat.ISO.DATE_TIME)
```

Comment:
Wouldn't better use a import?

Reply:
That's how it was generated, so I didn't want to mess with it or next time we generate it there would be a difference to investigate...